.gitlab-ci.yml

```diff
@@ -229,6 +229,26 @@ build:bleeding-edge:
     EXTRA_CONF: "-native-compiler yes -coqide opt"
     EXTRA_PACKAGES: "$COQIDE_PACKAGES"
 
+build:windows:
+  stage: test
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - dev\nsis\*.exe
+      - coq-opensource-archive-windows-*.zip
+    expire_in: 1 week
+  dependencies: []
+  tags:
+    - windows
+  before_script:
+    - SET ARCH=64
```

Comment:
It is for testing only that you only put "64" and you plan to also add "32" later?

Reply:
Yeah, I forgot to create a second job. Will fix.